src/components/SemiLiveProvider/index.js

```diff
@@ -1,44 +1,39 @@
-import React from "react";
+import React, { useState } from 'react';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import { TezosToolkit } from '@taquito/taquito';
 import { importKey, InMemorySigner } from '@taquito/signer';
-
+import styled from '@emotion/styled';
 
 class SemiLiveProvider extends LiveProvider {
-    constructor() {
-        super();
+  constructor() {
+    super();
 
-        this.onChange = code => {
-            // Override to prevent LiveProvider transpiling code on every change but
-            // keep the code. We will need it later.
-            console.log(code);
-            this.code = code;
-        };
-    }
+    this.onChange = code => {
+      // Override to prevent LiveProvider transpiling code on every change but
+      // keep the code. We will need it later.
+      console.log(code);
+      this.code = code;
+    };
+  }
 
-    UNSAFE_componentWillMount() {
-        // Override to prevent LiveProvider transpiling code on mount but
-        // keep the code. We will need it later.
-        this.code = this.props.code;
-    }
+  UNSAFE_componentWillMount() {
+    // Override to prevent LiveProvider transpiling code on mount but
+    // keep the code. We will need it later.
+    this.code = this.props.code;
+  }
 
-    componentDidUpdate() {
-        // Override to prevent LiveProvider transpiling code on update but
-        // keep the code. We will need it later.
-    }
+  componentDidUpdate() {
+    // Override to prevent LiveProvider transpiling code on update but
+    // keep the code. We will need it later.
+  }
 
-    run() {
-        
-        const { scope, transformCode, noInline } = this.props;
+  run() {
+    const { scope, transformCode, noInline } = this.props;
 
-        // The following piece of code provides additional functionality
-        // to the user code such as print function and key import
-        const code = `
-    let _printlnBuffer = "";
-    function print(value) {
-      _printlnBuffer += value + "\\n";
-      render(_printlnBuffer);
-    }
+    // The following piece of code provides additional functionality
+    // to the user code such as print function and key import
+    const code = `
+    
     Tezos.setProvider({ rpc: 'https://api.tez.ie/rpc/carthagenet' });
```

Comment:
RPC needs to be updated to delphi testnet https://api.tez.ie/rpc/delphinet

Reply:
Okay.